src/tekton/pipeline.ts

```diff
@@ -6,15 +6,28 @@
 import { TektonItem } from './tektonitem';
 import { TektonNode, Command } from '../tkn';
 import { Progress } from '../util/progress';
-import { window } from 'vscode';
+import { window, workspace } from 'vscode';
 import * as cliInstance from '../cli';
 import { cli } from '../cli';
 import { TknPipelineTrigger } from '../tekton';
 import { Trigger, PipelineContent } from './pipelinecontent';
+import pipelineViewLoader from '../view/pipeline/pipelineViewLoader';
+import { pipelineData } from './webviewstartpipeline';
 
 export class Pipeline extends TektonItem {
 
+
+  static checkWebViewStartPipeline(): boolean {
+    return workspace
+      .getConfiguration('vs-tekton')
+      .get<boolean>('StartPipeline');
+  }
+
   static async start(pipeline: TektonNode): Promise<string> {
+    if (Pipeline.checkWebViewStartPipeline()) {
```

Comment:
Maybe having separate command for webview wizard will be better? @siamaksade ?

Reply:
I think enabling webview from setting would be better.